src/test/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemDependenciesPageViewModelTest.kt

```diff
@@ -83,20 +94,34 @@ class SoftwareSystemDependenciesPageViewModelTest : ViewModelTest() {
         softwareSystem1.uses(externalSystem, "Uses", "REST")
 
         val viewModel = SoftwareSystemDependenciesPageViewModel(generatorContext, softwareSystem1)
-        assertThat(viewModel.dependenciesTable.bodyRows[0].columns[0])
+        assertThat(viewModel.dependenciesInboundTable.bodyRows[0].columns[0])
```

Comment:
I'll get this test updated but I do wonder if the test is no longer valid because the addSoftwareSystem(location, name,desc) method is now deprecated as the concept of Location.External and Location.Internal was removed from Structurizr with the deprecation of the enterprise concept.